pvlib/spectrum/mismatch.py

```diff
@@ -0,0 +1,213 @@
+    as shown in [1]_.
+
+    References
+    ----------
+    .. [1] Driesse, Anton, and Stein, Joshua. "Global Normal Spectral
+       Irradiance in Albuquerque: a One-Year Open Dataset for PV Research".
+       United States 2020. https://doi.org/10.2172/1814068.
+       https://www.osti.gov/servlets/purl/1814068.
+    '''
+    # Contributed by Anton Driesse (@adriesse), PV Performance Labs. Aug. 2022
+
+    SR_DATA = np.array([[ 290, 0.00],
+                        [ 350, 0.27],
+                        [ 400, 0.37],
+                        [ 500, 0.52],
+                        [ 650, 0.71],
+                        [ 800, 0.88],
+                        [ 900, 0.97],
+                        [ 950, 1.00],
+                        [1000, 0.93],
+                        [1050, 0.58],
+                        [1100, 0.21],
+                        [1150, 0.05],
+                        [1190, 0.00]]).transpose()
+
+    if wavelength is None:
+        resolution = 5.0
+        wavelength = np.arange(280, 1200 + resolution, resolution)
+
+    interpolator = interp1d(SR_DATA[0], SR_DATA[1],
+                            kind='cubic',
+                            bounds_error=False,
+                            fill_value=0.0,
+                            copy=False,
+                            assume_sorted=True)
+
+    sr = pd.Series(data=interpolator(wavelength), index=wavelength)
+
+    sr.index.name = 'wavelength'
+    sr.name = 'sr'
+
+    return sr
+
+
+def get_am15g(wavelength=None):
+    '''
+    Read the ASTM G173-03 AM1.5 global tilted spectrum, optionally interpolated
+    to the specified wavelength(s).
+
+    Parameters
+    ----------
+    wavelength: 1-D sequence of numeric, optional
+        The wavelengths at which the spectrum should be interpolated.
+        By default the 2002 wavelengths of the standard are returned. [nm]
+
+    Returns
+    -------
+    am15g: pandas.Series
+        The AM1.5g standard spectrum indexed by wavelength in nm. [W/m^2/nm]
+
+    Notes
+    -----
+    This function uses linear interpolation.  If the reference spectrum is too
+    coarsely interpolated, its integral may deviate from the standard value
+    of 1000.37 W/m^2.
+
+    The values in the data file provided with pvlib-python are copied from an
+    Excel file distributed by NREL, which is found here:
+    https://www.nrel.gov/grid/solar-resource/assets/data/astmg173.xls
+
+    More information about reference spectra is found here:
+    https://www.nrel.gov/grid/solar-resource/spectra-am1.5.html
+
+    References
+    ----------
+    .. [1] ASTM "G173-03 Standard Tables for Reference Solar Spectral
+        Irradiances: Direct Normal and Hemispherical on 37° Tilted Surface."
+    '''
+    # Contributed by Anton Driesse (@adriesse), PV Performance Labs. Aug. 2022
+
+    pvlib_path = pvlib.__path__[0]
+    filepath = os.path.join(pvlib_path, 'data', 'astm_g173_am15g.csv')
+
+    am15g = pd.read_csv(filepath, index_col=0, squeeze=True)
```

Comment:
This line issues a deprecation warning:
`pvlib-python/pvlib/spectrum/mismatch.py:115: FutureWarning: The squeeze argument has been deprecated and will be removed in a future version. Append .squeeze("columns") to the call to squeeze.`

Reply:
If only our present global squeeze could be deprecated and removed in a future version!